packages/types/src/electra/sszTypes.ts

```diff
@@ -181,9 +173,10 @@ export const BeaconBlockBody = new ContainerType(
     deposits: phase0Ssz.BeaconBlockBody.fields.deposits,
     voluntaryExits: phase0Ssz.BeaconBlockBody.fields.voluntaryExits,
     syncAggregate: altairSsz.BeaconBlockBody.fields.syncAggregate,
-    executionPayload: ExecutionPayload, // Modified in ELECTRA
+    executionPayload: ExecutionPayload,
     blsToExecutionChanges: capellaSsz.BeaconBlockBody.fields.blsToExecutionChanges,
     blobKzgCommitments: denebSsz.BeaconBlockBody.fields.blobKzgCommitments,
+    executionRequests: ExecutionRequests, // New in ELECTRA:EIP7251
```

Comment:
`BlindedBeaconBlockBody` needs to be updated with `execution_requests_root`, see [here](https://github.com/ethereum/builder-specs/pull/101/files#diff-2be1000fe5e3e5a9826d0c753b4acbaa496eb07148ef9b5698b8e3bc660fef89R57)

will be required to pass blinded data structures test
https://github.com/ChainSafe/lodestar/blob/bb40ef7eb77f9441ec66ab8f1fdf6664240bb1a3/packages/types/test/unit/blinded.test.ts#L6